recipes-extended/lighttpd/lighttpd_1.4.%.bbappend

```diff
@@ -38,4 +43,8 @@ do_install_append() {
     install -m 0644 ${WORKDIR}/wifi-configurator.css ${D}/www/pages/styles/
     install -m 0644 ${WORKDIR}/admin.css ${D}/www/pages/styles/
     install -m 0644 ${WORKDIR}/handler.js ${D}/www/pages/js/
+    install -m 0644 ${WORKDIR}/web.html ${D}/www/pages/
```

Comment:
Please, use single `install -m ... -D ... -t ... ` to install files into a target directory and to create this dir.